### PR TITLE
1211 rename first and last name fields

### PR DESCRIPTION
**Pull request by sebastien-comeau**
Self-explanatory

- Rename firstName to givenName
- Rename lastName to surname

[1211 - Rename PassportStatusEntity's First, LastName fields ](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1211/)

### CODE REVIEW

File: src/main/resources/db-migrations/postgresql/v0.1-[vendor]-schema-init.sql [hunk 1]
```diff
@@ -4,16 +4,24 @@ CREATE FUNCTION remove_diacritics(text)
...
+(remove_diacritics
+(given_name)));
 
-DROP INDEX ix_last_name;
-CREATE INDEX ix_last_name ON passport_status(lower(remove_diacritics(last_name)));
\ No newline at end of file
+DROP INDEX ix_surname;
+CREATE INDEX ix_surname ON passport_status(lower
+(remove_diacritics
+(surname)));
```
Comment (gregory-j-baker) [inline src/main/resources/db-migrations/postgresql/v0.1-[vendor]-schema-init.sql:27]: I really dislike the formatting of this file.. can you revert the formatting to what it was before?

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]
Your SQL formatter is sick.. and not in the good way.

File: src/main/resources/db-migrations/postgresql/v0.1-[vendor]-schema-init.sql (hunk 1 above)
Comment (sebastien-comeau) [reply src/main/resources/db-migrations/postgresql/v0.1-[vendor]-schema-init.sql:27]: Yark! Will fix!

Comment (sebastien-comeau) [reply src/main/resources/db-migrations/postgresql/v0.1-[vendor]-schema-init.sql]: Fixed!

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**